content/_includes/site/ccs.njk

```diff
@@ -9,19 +10,23 @@
     {% call forms.fieldset(
       legend='Color Mode',
       name='ccs-modes',
-      attrs={'data-radio': 'group'}
+      attrs={'data-group': 'buttons'}
     ) %}
       {% for mode in ['light', 'dark', 'auto'] %}
-        {{ forms.radio(
-          label=mode,
+
+        <button {{ toggle.btn(
           id=[mode, 'mode'] | join('-'),
           group='ccs-modes',
-          attrs={'data-ccs-colors': true if (mode == 'auto') else mode},
-          input_attrs={
-            'data-ccs-input': [mode, 'mode'] | join('-'),
-            'checked': true if (mode == 'auto') else false
+          attrs={
+            'data-ccs-colors': true if (mode == 'auto') else mode,
+            'data-ccs-input': id,
+            'aria-pressed': true if (mode == 'auto') else false,
+            'data-toggle-synced': 'true' if synced else 'false'
           }
         ) }}
+        >
+          {{ mode }}
+        </button>
```

Comment:
@jgerigmeyer This doesn't really work and I'd love some help on getting this functional before I style it as the markup may change based on how far off I am on this :) 

The goal was to replace the radio buttons with actual buttons (for reasons linked in the Trello card (https://lea.verou.me/2022/07/button-group/
https://tink.uk/perceived-affordances-and-the-functionality-mismatch/)

And then style them a bit differently. 

I know I hijacked the toggle macro, but I am pretty sure we weren't using that (yet?) and I wanted a place to start.

Reply:
Rather than updating this in OddSite, I think we should first adjust CCS to allow using buttons instead of just radio inputs.